packages/loaders/src/elements/Spinner.tsx

```diff
@@ -64,6 +64,9 @@ export interface ISpinnerProps extends React.HTMLAttributes<SVGSVGElement> {
   delayMS?: number;
 }
 
+/**
+ * @extends HTMLAttributes<SVGSVGElement>
```

Comment:
I think `SVGAttributes<SVGSVGElement>` is preferable, and the `interface` should be modified accordingly.